{
  "uid" : "568a1bfd0a3ead30",
  "name" : "Verify text in links in sections",
  "fullName" : "tests.used_resources_page_test.TestUsedResourcesPageForAuthorizedText#test_ur_02_04_verify_text_in_links",
  "historyId" : "766e01b17011752059bd2f6f8d4ac915",
  "time" : {
    "start" : 1728611061654,
    "stop" : 1728611061654,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x56523451002a <unknown>\n#1 0x5652341f65e0 <unknown>\n#2 0x5652341ee7eb <unknown>\n#3 0x5652341dea09 <unknown>\n#4 0x5652341e0744 <unknown>\n#5 0x5652341decbd <unknown>\n#6 0x5652341de52f <unknown>\n#7 0x5652341de422 <unknown>\n#8 0x5652341dc493 <unknown>\n#9 0x5652341dcada <unknown>\n#10 0x5652341f8f41 <unknown>\n#11 0x56523428a8c5 <unknown>\n#12 0x56523426ab22 <unknown>\n#13 0x565234289d7d <unknown>\n#14 0x56523426a8c3 <unknown>\n#15 0x5652342386b3 <unknown>\n#16 0x56523423968e <unknown>\n#17 0x5652344daa2b <unknown>\n#18 0x5652344de9b1 <unknown>\n#19 0x5652344c7225 <unknown>\n#20 0x5652344df532 <unknown>\n#21 0x5652344ac38f <unknown>\n#22 0x5652344fef28 <unknown>\n#23 0x5652344ff0f3 <unknown>\n#24 0x56523450ee7c <unknown>\n#25 0x7fed89894ac3 <unknown>",
  "statusTrace" : "driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"88e1a3457ad8acb59dabd5275d7b9a50\")>\n\n    @pytest.fixture()\n    @allure.step(f'Open page: {MainPageLinks.URL_MAIN_PAGE}')\n    def main_page_open(driver):\n>       driver.get(MainPageLinks.URL_MAIN_PAGE)\n\ntests/conftest.py:22: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:353: in get\n    self.execute(Command.GET, {\"url\": url})\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:344: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7f3bf88f7740>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"unknown error\",\"message\":\"unknown error: net::ERR_CONNECTION_REFUSED\\\\n  ...\\\\n#23 0x5652344ff0f3 \\\\u003Cunknown>\\\\n#24 0x56523450ee7c \\\\u003Cunknown>\\\\n#25 0x7fed89894ac3 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\nE         (Session info: chrome=129.0.6668.89)\nE       Stacktrace:\nE       #0 0x56523451002a <unknown>\nE       #1 0x5652341f65e0 <unknown>\nE       #2 0x5652341ee7eb <unknown>\nE       #3 0x5652341dea09 <unknown>\nE       #4 0x5652341e0744 <unknown>\nE       #5 0x5652341decbd <unknown>\nE       #6 0x5652341de52f <unknown>\nE       #7 0x5652341de422 <unknown>\nE       #8 0x5652341dc493 <unknown>\nE       #9 0x5652341dcada <unknown>\nE       #10 0x5652341f8f41 <unknown>\nE       #11 0x56523428a8c5 <unknown>\nE       #12 0x56523426ab22 <unknown>\nE       #13 0x565234289d7d <unknown>\nE       #14 0x56523426a8c3 <unknown>\nE       #15 0x5652342386b3 <unknown>\nE       #16 0x56523423968e <unknown>\nE       #17 0x5652344daa2b <unknown>\nE       #18 0x5652344de9b1 <unknown>\nE       #19 0x5652344c7225 <unknown>\nE       #20 0x5652344df532 <unknown>\nE       #21 0x5652344ac38f <unknown>\nE       #22 0x5652344fef28 <unknown>\nE       #23 0x5652344ff0f3 <unknown>\nE       #24 0x56523450ee7c <unknown>\nE       #25 0x7fed89894ac3 <unknown>\n\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py:229: WebDriverException",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "driver",
    "time" : {
      "start" : 1728611061655,
      "stop" : 1728611062423,
      "duration" : 768
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "main_page_open",
    "time" : {
      "start" : 1728611062423,
      "stop" : 1728611062847,
      "duration" : 424
    },
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x56523451002a <unknown>\n#1 0x5652341f65e0 <unknown>\n#2 0x5652341ee7eb <unknown>\n#3 0x5652341dea09 <unknown>\n#4 0x5652341e0744 <unknown>\n#5 0x5652341decbd <unknown>\n#6 0x5652341de52f <unknown>\n#7 0x5652341de422 <unknown>\n#8 0x5652341dc493 <unknown>\n#9 0x5652341dcada <unknown>\n#10 0x5652341f8f41 <unknown>\n#11 0x56523428a8c5 <unknown>\n#12 0x56523426ab22 <unknown>\n#13 0x565234289d7d <unknown>\n#14 0x56523426a8c3 <unknown>\n#15 0x5652342386b3 <unknown>\n#16 0x56523423968e <unknown>\n#17 0x5652344daa2b <unknown>\n#18 0x5652344de9b1 <unknown>\n#19 0x5652344c7225 <unknown>\n#20 0x5652344df532 <unknown>\n#21 0x5652344ac38f <unknown>\n#22 0x5652344fef28 <unknown>\n#23 0x5652344ff0f3 <unknown>\n#24 0x56523450ee7c <unknown>\n#25 0x7fed89894ac3 <unknown>\n\n",
    "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 902, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 22, in main_page_open\n    driver.get(MainPageLinks.URL_MAIN_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
    "steps" : [ {
      "name" : "Open page: https://brainup.site/",
      "time" : {
        "start" : 1728611062424,
        "stop" : 1728611062846,
        "duration" : 422
      },
      "status" : "broken",
      "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x56523451002a <unknown>\n#1 0x5652341f65e0 <unknown>\n#2 0x5652341ee7eb <unknown>\n#3 0x5652341dea09 <unknown>\n#4 0x5652341e0744 <unknown>\n#5 0x5652341decbd <unknown>\n#6 0x5652341de52f <unknown>\n#7 0x5652341de422 <unknown>\n#8 0x5652341dc493 <unknown>\n#9 0x5652341dcada <unknown>\n#10 0x5652341f8f41 <unknown>\n#11 0x56523428a8c5 <unknown>\n#12 0x56523426ab22 <unknown>\n#13 0x565234289d7d <unknown>\n#14 0x56523426a8c3 <unknown>\n#15 0x5652342386b3 <unknown>\n#16 0x56523423968e <unknown>\n#17 0x5652344daa2b <unknown>\n#18 0x5652344de9b1 <unknown>\n#19 0x5652344c7225 <unknown>\n#20 0x5652344df532 <unknown>\n#21 0x5652344ac38f <unknown>\n#22 0x5652344fef28 <unknown>\n#23 0x5652344ff0f3 <unknown>\n#24 0x56523450ee7c <unknown>\n#25 0x7fed89894ac3 <unknown>\n\n",
      "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 22, in main_page_open\n    driver.get(MainPageLinks.URL_MAIN_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "driver",
        "value" : "<selenium.webdriver.chrome.webdriver.WebDriver (session=\"88e1a3457ad8acb59dabd5275d7b9a50\")>"
      } ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  }, {
    "name" : "clear_allure_results_folder",
    "time" : {
      "start" : 1728610900918,
      "stop" : 1728610900918,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "afterStages" : [ {
    "name" : "driver::0",
    "time" : {
      "start" : 1728611062881,
      "stop" : 1728611062943,
      "duration" : 62
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "epic",
    "value" : "Test Used Resources Page"
  }, {
    "name" : "parentSuite",
    "value" : "tests"
  }, {
    "name" : "suite",
    "value" : "used_resources_page_test"
  }, {
    "name" : "subSuite",
    "value" : "TestUsedResourcesPage"
  }, {
    "name" : "host",
    "value" : "fv-az740-570"
  }, {
    "name" : "thread",
    "value" : "2375-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.used_resources_page_test"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 2,
        "skipped" : 0,
        "passed" : 30,
        "unknown" : 0,
        "total" : 32
      },
      "items" : [ {
        "uid" : "4f6897b898a03d6d",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/58//#testresult/4f6897b898a03d6d",
        "status" : "passed",
        "time" : {
          "start" : 1728525370569,
          "stop" : 1728525371341,
          "duration" : 772
        }
      }, {
        "uid" : "7e2d373639060699",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/57//#testresult/7e2d373639060699",
        "status" : "passed",
        "time" : {
          "start" : 1728439200362,
          "stop" : 1728439201135,
          "duration" : 773
        }
      }, {
        "uid" : "78bd00b6fcd93587",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/56//#testresult/78bd00b6fcd93587",
        "status" : "passed",
        "time" : {
          "start" : 1728352644192,
          "stop" : 1728352644984,
          "duration" : 792
        }
      }, {
        "uid" : "d62a527c9ef05246",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/55//#testresult/d62a527c9ef05246",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/groups",
        "time" : {
          "start" : 1728267766078,
          "stop" : 1728267766078,
          "duration" : 0
        }
      }, {
        "uid" : "2f65e0a57c12c280",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/54//#testresult/2f65e0a57c12c280",
        "status" : "passed",
        "time" : {
          "start" : 1728180179634,
          "stop" : 1728180180427,
          "duration" : 793
        }
      }, {
        "uid" : "ca6012f023fab901",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/53//#testresult/ca6012f023fab901",
        "status" : "passed",
        "time" : {
          "start" : 1728093478568,
          "stop" : 1728093479410,
          "duration" : 842
        }
      }, {
        "uid" : "1e56fcb44ecef00c",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/52//#testresult/1e56fcb44ecef00c",
        "status" : "passed",
        "time" : {
          "start" : 1728007184189,
          "stop" : 1728007185034,
          "duration" : 845
        }
      }, {
        "uid" : "d17b2ac5574c06ca",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/51//#testresult/d17b2ac5574c06ca",
        "status" : "passed",
        "time" : {
          "start" : 1727921050195,
          "stop" : 1727921051051,
          "duration" : 856
        }
      }, {
        "uid" : "d354651343b2327d",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/50//#testresult/d354651343b2327d",
        "status" : "passed",
        "time" : {
          "start" : 1727834353209,
          "stop" : 1727834353998,
          "duration" : 789
        }
      }, {
        "uid" : "ff81fa579e45e48f",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/49//#testresult/ff81fa579e45e48f",
        "status" : "passed",
        "time" : {
          "start" : 1727748508649,
          "stop" : 1727748509435,
          "duration" : 786
        }
      }, {
        "uid" : "4b838105256e0a06",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/48//#testresult/4b838105256e0a06",
        "status" : "passed",
        "time" : {
          "start" : 1727661761929,
          "stop" : 1727661762720,
          "duration" : 791
        }
      }, {
        "uid" : "e1279a5d8e4baaf0",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/47//#testresult/e1279a5d8e4baaf0",
        "status" : "passed",
        "time" : {
          "start" : 1727575681795,
          "stop" : 1727575682575,
          "duration" : 780
        }
      }, {
        "uid" : "4bfebe3d839e2a51",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/46//#testresult/4bfebe3d839e2a51",
        "status" : "passed",
        "time" : {
          "start" : 1727489016563,
          "stop" : 1727489017348,
          "duration" : 785
        }
      }, {
        "uid" : "27c2eaba3f643e87",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/45//#testresult/27c2eaba3f643e87",
        "status" : "passed",
        "time" : {
          "start" : 1727402717825,
          "stop" : 1727402718682,
          "duration" : 857
        }
      }, {
        "uid" : "9f6c78387e4d83d5",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/44//#testresult/9f6c78387e4d83d5",
        "status" : "passed",
        "time" : {
          "start" : 1727316273534,
          "stop" : 1727316274373,
          "duration" : 839
        }
      }, {
        "uid" : "c4b2a94d0ca5fdfb",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/43//#testresult/c4b2a94d0ca5fdfb",
        "status" : "passed",
        "time" : {
          "start" : 1727229882397,
          "stop" : 1727229883190,
          "duration" : 793
        }
      }, {
        "uid" : "fe60ce3470147c44",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/42//#testresult/fe60ce3470147c44",
        "status" : "passed",
        "time" : {
          "start" : 1727143207829,
          "stop" : 1727143208623,
          "duration" : 794
        }
      }, {
        "uid" : "47360d77b53049fb",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/41//#testresult/47360d77b53049fb",
        "status" : "passed",
        "time" : {
          "start" : 1727056640288,
          "stop" : 1727056641057,
          "duration" : 769
        }
      }, {
        "uid" : "eb9bc717a3d5fd24",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/40//#testresult/eb9bc717a3d5fd24",
        "status" : "passed",
        "time" : {
          "start" : 1726970463995,
          "stop" : 1726970464764,
          "duration" : 769
        }
      }, {
        "uid" : "546f2e647c9982c6",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/39//#testresult/546f2e647c9982c6",
        "status" : "passed",
        "time" : {
          "start" : 1726883641854,
          "stop" : 1726883642621,
          "duration" : 767
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "568a1bfd0a3ead30.json",
  "parameterValues" : [ ]
}